src/service/url-replacements-impl.js

```diff
@@ -313,6 +313,20 @@ export class GlobalVariableSource extends VariableSource {
       return Date.now();
     });
 
+    //Returns the human readable timestamp in format of 2011-01-01 11:11:11.
```

Comment:
`toISOString()` is what you want.

We should not do this `FORMATTED` thing. We need to at the very least adhere to some standard and put that in the name. Why is ISO bad?